daemon/containerd/image_commit.go

```diff
@@ -143,7 +144,7 @@ func generateCommitImageConfig(baseConfig ocispec.Image, diffID digest.Digest, o
 		},
 		History: append(baseConfig.History, ocispec.History{
 			Created:    &createdTime,
-			CreatedBy:  "", // FIXME(ndeloof) ?
+			CreatedBy:  strings.Join(opts.ContainerConfig.Cmd, " "),
```

Comment:
For ease of review – this is the same transformation as we do in other places https://github.com/moby/moby/blob/06221297bf58ff1ee7b9a546e8feabf042fa8a49/image/image.go#L229